src/style.css

```diff
@@ -1 +1,169 @@
-/* empty source */
+/* #region Normalization*/
+body {
+  margin: 0;
+  background: rgba(255, 255, 255, 1);
+}
+/*#endregion*/
+
+.mainCard {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+}
+/* #region image */
+
+.image {
+  /* display: inline-block; */
+  margin: 32px 19px 36px;
+}
+
+.photo {
+  height: 134px;
+  /* width: 160px; */
+
+}
+/* #endregion */
+
+/*#region description */
+
+.mainCard__description {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  flex-wrap: nowrap;
+  width: 166px;
+  margin: 0 16px 16px;
+}
+
+.mainCard__model {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;

```

Comment:
there is top margin for discription of 40 px , where you have stylled it?
<img width="434" alt="image" src="https://user-images.githubusercontent.com/100297341/201103036-2fd6384f-700a-486e-9560-377ffefe0afe.png">

Reply:
not implemented